Sources/AltSwiftUI/Source/Views/Collections/HStack.swift

```diff
@@ -83,8 +83,7 @@ extension HStack: Renderable {
         view.spacing = spacing
     }
     
-    private var subviewIsEquallySpaced: (View) -> Bool {
-        { view in
+    private var subviewIsEquallySpaced: (View) -> Bool { { view in
```

Comment:
Is this a rule in swift lint? I think it actually looks better with it starting in the next line as this is not a trailing closure but a return type.

Reply:
I think some SwiftLint rule is autocorrecting this. I haven't fixed this myself manually.